pkg/scheduler/apis/config/v1beta2/defaults.go

```diff
@@ -211,7 +211,11 @@ func SetDefaults_VolumeBindingArgs(obj *v1beta2.VolumeBindingArgs) {
 func SetDefaults_PodTopologySpreadArgs(obj *v1beta2.PodTopologySpreadArgs) {
 	if feature.DefaultFeatureGate.Enabled(features.DefaultPodTopologySpread) {
 		if obj.DefaultingType == "" {
-			obj.DefaultingType = v1beta2.SystemDefaulting
+			if len(obj.DefaultConstraints) != 0 {
```

Comment:
why revert this? It was intended. Defaults should be independent of other fields.

Reply:
Oops, good catch, I missed this when reviewing.